src/components/backdrop/backdrop.js

```diff
@@ -66,6 +71,13 @@ angular
         }
       });
 
+      function resize() {
```

Comment:
Yes, but on old version. It is fixed on iOS 8 and above as far as I know. Lower versions if still important can be targeted with media queries: https://gist.github.com/BenMorel/e9e34c08360ebbbd0634 (link taken from caniuse)